client/src/pages/WhatIsAutism/WhatIsAutism.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import ReactPlayer from "react-player/youtube";
+import { BsPlayCircleFill } from "react-icons/bs";
+import whatIsAutismImg from "../../assets/images/what-is-autism.jpeg";
+import whatIsAutismVideoImg from "../../assets/images/what-is-autism-video.jpg";
+import FooterNav from "../Home/FooterNav/FooterNav";
+import FooterIcon from "../Home/FooterIcon/FooterIcon";
+import FooterLogo from "../Home/FooterLogo/FooterLogo";
+import "./WhatIsAutism.css";
+
+function WhatIsAutism() {
+	return (
+		<div className="what-is-autism-container">
+			<img
+				className="hero-image"
+				src={whatIsAutismImg}
+				alt="A woman and a girl are talking"
+			></img>
+			<div className="main-container">
+				<div className="discription-container">
+					<div className="discription">
+						Autism or Autism Spectrum Disorder is not an illness or a disease.
+						<br />
+						Autism is not a medical condition with treatments or a “cure”. It
+						<br />
+						means your brain works in a different way from other people.
+						<br /> It’s something you’re born with or first appears when you’re
+						very <br />
+						young.
+						<br />
+						Everyone with autism is different, that’s what makes it a spectrum.
+						<br />
+						Some people may need little or no support. Other may need extra
+						<br />
+						help to manage certain things such as daily tasks, activities, jobs
+						or
+						<br />
+						relationships.
+					</div>
+					<div className="more-info-autism">
+						For more info and a guide about Autism <br />
+						visit the NHS website:
+						<a
+							href="https://www.nhs.uk/conditions/autism/"
+							target="_blank"
+							rel="noreferrer"
+						>
+							https://www.nhs.uk/conditions/autism/
+						</a>
+					</div>
+				</div>
+				<div className="video-container">
+					<div className="video-wrapper">
+						<ReactPlayer
+							className="player"
```

Comment:
i would suggest you  add a controls prop to your react player so the video runs with the controls underneath it

Reply:
👍 second that :) 
here is the [example](https://github.com/filkovsp/web-fs/blob/develop/client/src/Components/ViewerFactory.js#L83)

Reply:
Oh, I couldn't find that by myself, thank you!!!